_includes/schedule

```diff
@@ -7,75 +7,102 @@
 <p></p>
 
 <style>
-    .schedule-table-heading {
-        display: inline;
-        font-weight: bold;
-        font-size: 20px;
-        color: #999999;
-        padding:0 0 20px 0;
-    }
-    
-    .schedule-table-timecol {
-        padding:0 50px 0 50px;
-        display:inline;
-    }
-    
-    .schedule-table-eventcol {
-        display:inline;
-        display:inline-block;
-        inline-size: 300px;
-        font-size:14px;
-    }
-    
-    .schedule-table-contentcol {
-        display:inline;
-        display:inline-block;
-        inline-size: 250px;
-        font-size:14px;
-        line-height: normal;
-    }
-    
-    .schedule-table-row-even {
-        display:block;
-        width:800px;
-        background-color: #EEEEEE;
-        padding:10px;
-    }
-    
-    .schedule-table-row-odd {
-        display:block;
-        width:800px;
-        padding:10px;
-    }
-</style>
+  .schedule-table-heading {
+    display: inline;
+    font-weight: bold;
+    font-size: 20px;
+    color: #999999;
+    padding: 0 0 20px;
+  }
+
+  .schedule-table-timecol {
+    padding: 0 50px;
+    display: inline;
+  }
 
-<div style="display:block; width:900px; padding:20px; border:solid 4px #CCCCCC;">
+  .schedule-table-eventcol {
+    display: inline;
+    display: inline-block;
+    inline-size: 350px;
+    font-size: 14px;
+  }
 
-    <div class="schedule-table-heading" style="margin-left:57px; display:inline-block; inline-size:100px;">Time</div>
-    <div class="schedule-table-heading" style="display:inline-block; inline-size:295px;">Event</div>
-    <div class="schedule-table-heading">Content</div>
```

Comment:
@maximilianigl did you delete the table heading on purpose?

Reply:
Ah no, I didn't, sorry. I only wanted to change the width to 1000 px.

Reply:
I'll fix it.